skopt/optimizer/optimizer.py

```diff
@@ -236,46 +256,59 @@ def tell(self, x, y, fit=True):
                 warnings.simplefilter("ignore")
                 est.fit(self.space.transform(self.Xi), self.yi)
 
+            if hasattr(self, "next_xs_") and self.acq_func == "gp_hedge":
+                self.gains_ -= est.predict(np.vstack(self.next_xs_))
```

Comment:
@MechCoder In the original paper (algorithm 2 line 9), the posterior predictive mean is an addition to the gains instead of a subtraction. Is this a mistake or something related to the switch from maximization to minimization? I'm a little uncertain.

Reply:
To add on this a little bit, the outside multi-armed bandit problem functions to choose the best method among the three. Therefore, whichever next point is chosen, the gains shall enhance (instead of diminish) the importance of that exact arm?

Reply:
Should we create an issue to discuss this?

Reply:
@betatim Excuse me for the late reply. It could be a false alarm, since I did a [test](https://github.com/kejiashi/optimization-dump-truck/blob/master/hedge_test.ipynb) and found no issue with the original code. But I think some of my understandings of the code need to be corrected and discussed. Let me open a new issue.

Reply:
thanks :)